app/views/searches/advanced.html.erb

```diff
@@ -11,18 +19,18 @@
 
   <dl class="search-fields">
     <dt><%= label_tag :name, t(".name"), class: "form__label" %></dt>
-    <dd><%= text_field_tag :name, "", placeholder: "active OR action", class: "form__input"%></dd>
+    <dd><%= text_field_tag :name, "", placeholder: "active OR action", class: "form__input", data: { search_target: "attribute", action: "input->search#input keydown.enter->search#submit" } %></dd>
```

Comment:
I'm total Stimulus.js noob, probably I'm missing something, but wouldn't be 1 event of "change" (instead input and keydown.enter) enough?

Reply:
sorry, I mean "input" only

Reply:
ahh, I do follow now, looks ok